meta-lmp-base/recipes-support/bluetooth-attach/bluetooth-attach/hciattach-custom.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
`bash` is GPL so is better to have `sh` that uses busybox

Reply:
@quaresmajose bash is my possession :) but you're right. Replacing...

Reply:
some syntax below won't work with busybox's shell. Apart from that it looks OK to me.